app/src/main/java/com/kamron/pogoiv/logic/IVCombination.java

```diff
@@ -38,6 +38,21 @@ public boolean equals(Object o) {
     }
 
     /**
+     * get the highest of the stats in the iv combination
+     * for example, in the iv combination 7 5 11
+     * this method would return 11
+     * @return a number between 0 and 15 which is the highest stat in this combination
+     */
+    public int getHighestStat(){
+        int high = 0;
+
+        if (att > high) high = att;
+        if (def > high) high = def;
+        if (sta > high) high = sta;
+
+        return high;
+    }
+    /**
```

Comment:
Below in `getHighestStatSignature` we have already `Collections.max(Arrays.asList(att, def, sta))`. Let's not add another implementation of that.